packages/client-engine-runtime/jest.config.js

```diff
@@ -1,3 +1,5 @@
+require('dotenv').config({ path: '../../.db.env' })
```

Comment:
Why do we need the connection strings in these tests? I don't think we do any real IO here, there are only unit tests

Reply:
That makes sense, e.g., for the functional tests in `packages/client`. What I don't understand is why this is necessary for the tests to work inside IDE in this package specifically.

Reply:
You're right, it is not needed here, since there is no real DB access. Removed.